requirements.txt

```diff
@@ -4,3 +4,4 @@ six>=1.12.0
 tabulator==1.53.5
 Unidecode==1.0.22
 python-dateutil>=2.8.2
+chardet==5.2.0
```

Comment:
Might be better to make this a range. We could handle earlier versions and probably later patch versions.

Reply:
unsure, also Unidecode is 2018 for 1.0.22. they now have 1.3.7

Reply:
2018-11-19	unidecode 1.0.23 
 Drop support for Python 2.6 and 3.3 (thanks to Jon Dufresne)
 
 I think we can now move to latest since we are not locked to 2.x anymore.

Reply:
Coming from a maven background, I'm opinionated to locked versions. We could go with updates but can cause side effects if not picked up.

Also dependabot exists and will tell us to update and kick off testing etc.

Reply:
We could at least allow any 5.2.x version so we get security patches.